zabbix/files/default/etc/zabbix/zabbix_agentd.conf.jinja

```diff
@@ -127,8 +127,10 @@ LogRemoteCommands={{ settings.get('logremotecommands', defaults.logremotecommand
 # Mandatory: yes, if StartAgents is not explicitly set to 0
 # Default:
 # Server=
-{% if settings.get('server', defaults.get('server', False)) %}
+{% if settings.get('server', defaults.get('server', False)) is list %}
```

Comment:
`is list` - there is no such test in Jinja https://jinja.palletsprojects.com/en/2.11.x/templates/#builtin-tests can you give a reference for this construction?

Methods known to me:
* `{% if var is iterable and (var is not string and var is not mapping) %}` ref. https://stackoverflow.com/a/39453833/2444141
* Using Salt specific filter `{% if var|is_list %}` ref. https://docs.saltproject.io/en/latest/topics/jinja/index.html#is-list

Reply:
wouldn't the is_list call the same code? i didn't find any documentation for that explicit format, but it's difficult to google :)
there are usages like this in saltstack-formulas though.  I can change it to the |is_list if you're more comfortable with that, but this variant imho looks better

Reply:
It's not about comfort, it's about using documented features vs undocumented ones. `is_list` is a [custom jinja filter](https://github.com/saltstack/salt/blob/0b2a5613b345f17339cb90e60b407199b3d26980/salt/utils/data.py#L1005), `is list` - how it's working? Can you give a reference to Salt or Jinja code which will explain why this construction even working?

Reply:
i've changed it to the "| is_list" instead, sorry i have no explicit documentation for the previous variant  ... it just works :)